Makefile

```diff
@@ -123,6 +131,17 @@ $(OC) oc: $(TOOLS_DIR)
 		$(OC) version | grep -q $${version##v} ;\
 	}
 
+.PHONY: jsonnet-tools
+$(JSONNET) jsonnet-tools: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(JSONNET) && -f $(JSONNETFMT) && -f $(JSONNET_BUNDLER) && -f $(GOJSONTOYAML) ]] && exit 0 ;\
```

Comment:
it is better to check individually ... e.g.

```Makefile
.PHONY: jsonnet-tools
$(JSONNET) $(JSONNETFMT) $(JSONNET_BUNDLER) $(GOJSONTOYAML) jsonnet-tools: $(TOOLS_DIR)
	@{ \
			set -ex ;\
			export GOBIN=$(TOOLS_DIR) ;\
			[[ -f $(JSONNET) ]] ||  go install github.com/google/go-jsonnet/cmd/jsonnet@latest ;\
			[[ -f $(JSONNETFMT) ]] || go install github.com/google/go-jsonnet/cmd/jsonnetfmt@latest ;\
			[[ -f $(JSONNET_BUNDLER) ]] || go install github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest ;\
			[[ -f $(GOJSONTOYAML) ]] || go install github.com/brancz/gojsontoyaml@latest ;\
		}
```

Do you also need `-a` when installing jsonnet-bundler ? Could you please comment/doc why  that is required?

Reply:
The reason I used `-a` was because per readme of [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler#install) but it works without that when I tested I will just remove that flag